absearch/tests/test_server.py

```diff
@@ -30,6 +32,29 @@ def test_info():
     assert info['version'] == __version__
 
 
+def test_swagger():
+    app = get_app()
+    # test the APIs
+    res = app.get('/__api__')
+
+    # make sure it's compliant
+    parser = SwaggerParser(swagger_dict=yaml.load(res.body))
+    spec = parser.specification
+
+    assert spec['info']['version'] == __version__
+    assert spec['schemes'] == ['https']
+    assert spec['host'] == 'localhost:80'
+
+    # now testing that every GET endpoint is present
+    for path, items in spec['paths'].items():
+        for verb, options in items.items():
+            verb = verb.upper()
+            if verb != 'GET':
+                continue
+            statuses = [int(st) for st in options['responses'].keys()]
```

Comment:
the test wants to test that every endpoint is present, but in `statuses` you'll have `404`.